OpenOversight/manage.py

```diff
@@ -90,5 +92,49 @@ def link_officers_to_department():
     db.session.commit()
 
 
+@manager.command
+def bulk_add_officers(filename):
+    """Bulk adds officers."""
+    csvfile = csv.DictReader(open(filename))
+    departments = {}
+    n_created = 0
+    for line in csvfile:
+        department_id = line['department_id']
+        department = departments.get(department_id)
+        if line['department_id'] not in departments:
+            department = Department.query.filter_by(id=department_id).one_or_none()
+            if department:
+                departments[department_id] = department
+            else:
+                raise Exception('Department ID {} not found'.format(department_id))
+
+        # check for existing officer based on badge
+        if badge_exists(line['badge'], department_id):
+            print 'Skipping creation of existing badge id {} for department id {}'.format(line['badge'], department_id)
+            continue
+
+        # create officer
+        officer = Officer()
+        officer.department_id = department_id
+        officer.last_name = line['last_name']
+        officer.first_name = line['first_name']
+        officer.middle_initial = line['middle_initial']
+        officer.race = line['race']
+        officer.gender = line['gender']
+        officer.employment_date = line['employment_date']
+        officer.birth_year = line['birth_year']
+        db.session.add(officer)
+        db.session.commit()
```

Comment:
I believe this `commit()` will throw constraint-violation-related exceptions if the officer already exists; we might want to wrap the commit in a `try` and log the failed officer on exception? This would also have the advantage of making reimports closer to idempotent.